drivers/hyperv/hyperv.go

```diff
@@ -43,6 +50,7 @@ func NewDriver(hostName, storePath string) *Driver {
 		DiskSize:             defaultDiskSize,
 		MemSize:              defaultMemory,
 		CPU:                  defaultCPU,
+		WindowsVHDUrl:        mcnutils.ConfigGuest.GetVHDUrl(),
```

Comment:
is there a default value that minikube will download the VHD in case the user does not provide one.